### DISCUSSION TITLE
Use already existant Laravel login page instead of Statamic one

### DISCUSSION THREAD

**Discussion by rooselle** [statamic/cms#9855] [Q&A] [answered]
Hi,
I've installed Statamic into an existing Laravel app, which already has users and a login page. Statamic adds its own login page, and both pages work: I can log in with one or the other without any problem. _However_, if I access the control panel directly from the URL and I happen to not be logged in, I am redirected to the login page of Statamic. What I would like is to be redirected to the existing login page of my app... is there a way to do that? (And, by the way, to delete altogether the Statamic login page/forgot password page.) I didn't find anything in the doc, and I can see where the `login` route of Statamic is created and used, but not where I could intervene to tell Statamic to use my login page. Should I create a custom middleware? Is there a config somewhere to change?
Thanks in advance for your help :pray:

**Comment by duncanmcclean** [comment, marked as answer]
It's not possible, yet! 

However, there's an open PR which'll make this possible: https://github.com/statamic/cms/pull/8960

**Reply by rooselle** [reply]
Wow! Thanks for your prompt reply!! And for this great news :star_struck:
